### ISSUE TITLE
feat: Enable dataset can support fine-tuning

### ISSUE THREAD

**Issue by gyliu513** [langfuse/langfuse#449]
### Describe the feature you'd like to request

The current dataset https://langfuse.com/docs/datasets can only support benchmark test, it will be great to enable the dataset can also be used for fine tuning as well.

### Describe the solution you'd like to see

I can use the data in the dataset do some fine tuning work for my model.

### Contribute

- [ ] Yes, I can implement this and raise a PR

**Comment by marcklingen** [comment]
@gyliu513 would it be helpful to download a dataset similar to how observations can be downloaded?
https://langfuse.com/docs/export-and-fine-tuning#export-via-ui

**Comment by gyliu513** [comment]
Yes, I can download that, but just wondering is it possible to enable langfuse can do fine-tuning itself just like https://docs.promptlayer.com/why-promptlayer/fine-tuning ? Not sure if this is in your roadmap or not. If langfuse manily for analytics, maybe it will not cover this. comments? Thanks

**Comment by marcklingen** [comment]
@gyliu513 Thanks for sharing your thoughts and the link. I like the idea of running these jobs from within the Langfuse UI as this is totally doable based on the data that's in Langfuse today. I think we have higher-importance roadmap items right now but I'm totally open for PRs and reevaluate this in the future
Short-term, we could add a notebook on how to do this on your own as a quick solution.

**Comment by gyliu513** [comment]
@marcklingen a notebook is good for now, let me check if I have some cycle for this, thanks

**Comment by marcklingen** [comment]
Sounds good, happy to help if I can @gyliu513
